scripts/prepare_release.sh

```diff
@@ -144,7 +144,9 @@ create_release_branch() {
 	fi
 
 	eval git checkout -b "${release_branch}" "$mute"
-	eval git push -u origin "${release_branch}" "$mute"
+	if ! eval git push -u origin "${release_branch}" "$mute"; then
```

Comment:
Tested this with a dummy branch but my bash-fu is rusty so lmk if there's better way to do it.